### PR TITLE
Backend fixes

### PR DESCRIPTION
**Pull request by pohlinwei**

### CODE REVIEW

File: frontend/src/logic/offerlogic.ts [hunk 1]
```diff
@@ -61,41 +61,48 @@ const offerConverter = Object.freeze({
...
   >[] = [];
-  requestRef.forEach((offerSnapshot) => {
+  offersRef.forEach((offerSnapshot) => {
     temp.push(offerSnapshot);
   });
   await Promise.all(
     temp.map(async (offerSnapshot) => {
       try {
-        const offer = await offerConverter.fromFirestore(offerSnapshot);
+        const offer = offerConverter.fromFirestore(offerSnapshot);
```
Comment (EugeneTeu) [inline frontend/src/logic/offerlogic.ts]: this is not an async method for offer is it? Cause i recall for request we still need to make a call to get the task

**Review by EugeneTeu — COMMENTED** [review]
LGTM,

**Review by leafgecko — APPROVED** [review]

File: frontend/src/logic/offerlogic.ts (hunk 1 above)
Comment (pohlinwei) [reply frontend/src/logic/offerlogic.ts]: Nope. It isn't. 😅

**Comment by pohlinwei** [comment]
@EugeneTeu Done! Look at "logic/index.ts" for all the methods which you could use on the FE. (:

**Review by EugeneTeu — APPROVED** [review]

**Pull request merged**